src/hotspot/share/logging/logFileStreamOutput.cpp

```diff
@@ -115,18 +116,34 @@ bool LogFileStreamOutput::flush() {
   total += result;                                            \
 }
 
-int LogFileStreamOutput::write_internal(const LogDecorations& decorations, const char* msg) {
+int LogFileStreamOutput::write_internal_line(const LogDecorations& decorations, const char* msg, int msg_len) {
```

Comment:
```suggestion
int LogFileStreamOutput::write_internal_lines(const LogDecorations& decorations, const char* msg, int msg_len) {
```
Not sure we can't just fold this altogether in `write_internal`.

Reply:
I guess we can, but I liked the idea of separating the logic for each "step". In any case, I changed the function name as per your suggestion as it made it to look like it just printed one line